src/app/public/plugin-resources/image-resources-provider.ts

```diff
@@ -0,0 +1,8 @@
+import {
+  SkyAppLocaleInfo,
+  SkyLibResourcesProvider
+} from '@skyux/i18n';
+
+export class SkyImageResourcesProvider implements SkyLibResourcesProvider {
```

Comment:
Since this will be used by all modules in this library, we should name it `SkyMediaResourcesProvider`.